Day-II-Calculator/calculator/src/App.css

```diff
@@ -0,0 +1,14 @@
+.calculator-wrapper {
+    width: 100%;
+    min-height: 600px;
+    max-width: 450px;
```

Comment:
The styling will be perfect if you change the width to 400px. Otherwise, you'll need to change your button widths/height ratios.